hub/hub-api.yaml

```
@@ -1919,6 +1919,8 @@ components:
     User:
       type: object
       description: The user contact details and profile settings, including preferred authentication method, assigned roles, etc.
+      required:
+        - id
```

```suggestion
        - id
        - email
        - hasVerifiedEmail
        - isAdmin
        - isOwner
```
I think there is a few more fields, but just as a suggestion, let's check all non-null fields and add them here.

---

Done. Checked with the SQL table specification.